File: core/Core/Models/Notification.cs
using System;
using System.Collections.Generic;

namespace Core.Models;

public partial class Notification
{
    public int NotificationId { get; set; }

    public int? CustomerId { get; set; }

    public string? Message { get; set; }

    public DateTime? CreatedAt { get; set; }

    public bool? IsRead { get; set; }

    public virtual Customer? Customer { get; set; }
}

File: analytic/domain/notification/notification.py
from datetime import datetime

class Notification:
    def __init__(self,
                 notification_id: int,
                 customer_id: int | None,
                 message: str | None,
                 created_at: datetime | None,
                 is_read: bool | None):
        self.notification_id = notification_id
        self.customer_id = customer_id
        self.message = message
        self.created_at = created_at
        self.is_read = is_read